File: app/src/main/java/de/bahnhoefe/deutschlands/bahnhofsfotos/model/PhotoStation.java
package de.bahnhoefe.deutschlands.bahnhofsfotos.model;

import java.util.List;

import lombok.Builder;
import lombok.Value;

@Value
@Builder
public class PhotoStation {

    String country;

    String id;

    String title;

    double lat;

    double lon;

    String shortCode;

    boolean inactive;

    List<Photo> photos;

}

File: app/src/main/kotlin/de/bahnhoefe/deutschlands/bahnhofsfotos/model/PhotoStation.kt
package de.bahnhoefe.deutschlands.bahnhofsfotos.model

data class PhotoStation @JvmOverloads constructor(
    val country: String,
    val id: String,
    val title: String,
    val lat: Double,
    val lon: Double,
    var shortCode: String? = null,
    var inactive: Boolean = false,
    val photos: List<Photo> = listOf()
)